google_compute_engine/distro/tests/helpers_test.py

```diff
@@ -64,3 +64,84 @@ def testCallDhclient(self, mock_call, mock_exists):
     ]
 
     self.assertEqual(mocks.mock_calls, expected_calls)
+
+  @mock.patch('google_compute_engine.distro.helpers.subprocess.check_call')
+  def testCallHwclock(self, mock_call):
+    command = ['/sbin/hwclock', '--hctosys']
+    mock_logger = mock.Mock()
+
+    helpers.CallHwclock(mock_logger)
+    mock_call.assert_called_once_with(command)
+    expected_calls = [
```

Comment:
This fits on one line in under 80 characters; no need to split it up.

This also applies to the `expected_calls` below.